### DISCUSSION TITLE
pip install -q cython_bbox fails

### DISCUSSION THREAD

**Discussion by StefanoGarzoni** [roboflow/notebooks#104] [Q&A]
I'm trying to install this library on my project in anaconda3 and every time I try to install -q cython_bbox it gives me the following error:

error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [11 lines of output]
      running bdist_wheel
      running build
      running build_ext
      building 'cython_bbox' extension
      creating build
      creating build\temp.win-amd64-cpython-37
      creating build\temp.win-amd64-cpython-37\Release
      creating build\temp.win-amd64-cpython-37\Release\src
      "C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD - IC:\ProgramData\Anaconda3\envs\countingYolo\lib\site-packages\numpy\core\include -IC:\ProgramData\Anaconda3\envs\countingYolo\include -IC:\ProgramData\Anaconda3\envs\countingYolo\Include "- IC:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include" "-IC:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34. 31933\ATLMFC\include" "-IC:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0. 22000.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22000.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0 .22000.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22000.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include \10.0.22000.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" /Tcsrc/cython_bbox.c /Fobuild\temp.win-amd64-cpython-37 \Release\src/cython_bbox.obj -Wno-cpp
      cl : command line error D8021 : invalid numeric argument '/Wno-cpp'
      error: command 'C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for cython_bbox
ERROR: Could not build wheels for cython_bbox, which is required to install pyproject.toml-based projects

can someone help me please?
Thanks so much in advance!

**Comment by SkalskiP** [comment]
Here is a solution I found :) could you test it? 

![Screenshot 2023-04-27 at 13 06 52](https://user-images.githubusercontent.com/26109316/234844481-160105d0-f56a-4b97-a7f8-86d9999d69bc.png)

**Comment by StefanoGarzoni** [comment]
problem solved!
Thank you very much !
If I have other problems can I ask directly in this thread?
